python/src/Analytical_doc.i.in

```diff
@@ -9,7 +9,9 @@ event : :class:`~openturns.RandomVector`
     Failure event.
 physicalStartingPoint : sequence of float
     Starting point of the optimization algorithm, declared in the physical
-    space.
+    space (in case of single-start algorithm). In case of multi-start algorithm,
+    the starting points are directly defined in nearestPointAlgorithm in the 
+    Standard Space.
```

Comment:
this part is gone since physicalStartingPoint argument was deprecated

Reply:
fixed